Add AzCLI and zip to dockerfile (#15)
NarmathaBala/vsts-extension-retrospectives: CONTRIBUTING.md

3. Publish the newly created branch to the Retrospectives repo. Use this branch as your working branch.
4. Once you are ready to check-in, create a pull request against the 'master' branch. Link the Bug/Task that you are fixing/adding to the pull request. Reviewers will be added automatically.

> ## Considerations

Reviewer: @mindlessroman Adding the az cli & zip install to the Dockerfile is a good addition but I think the CONTRIBUTING.md changes may need to be reworked. This new section may give the impression that standing up a backend is required to contribute which is not the case. I like the list of dependencies (when not using the dev container) but would move to the "Backend" section. The "Backend" section may need some additional clarity on the fact that it's an optional requirement and is only needed for realtime updates. We can discuss more at stanup.

Reviewer 2: +1 to language around env_setup.sh. Good to differentiate backend set up from frontend by adding sections for them.

Author: Adding some context now

> Provided in this repo is the environement setup script. Running `env_setup.sh` spins up inidivudal

Reviewer 2: ```suggestion
Provided in this repo is the environment setup script. Running `env_setup.sh` spins up individual
```

Author: I was on a role with those typos!

Reviewer 3: > I was on a role with those typos!

I was on a roll with those typos!

instances of the different Azure components for this extension. If you do not have all the packages
and utilities installed already, it can slow down your development :(

> *The easiest solution?* This repo provides a Dockerfile and a devcontainer setup. This creates a simple,

Reviewer 2: ```suggestion
*The easiest solution* This repo provides a Dockerfile and a devcontainer setup. This creates a simple,
```

containerized development environment for contribuing to this extension. Follow along with [this video](https://www.youtube.com/watch?v=GwsscfcAjg8)

> by @polatengin that talks about the different ways to confirgure your environment.

Reviewer 2: ```suggestion
by @polatengin that talks about the different ways to configure your environment.
```

CollaborationStateServiceUrl value to the App Service URL -
`https://<RESOURCE_NAME_SUFFIX>.azurewebsites.net` and redeploy the extension.

> #### Considerations for the Backend

Reviewer: @mindlessroman I think it's safe to remove this "Considerations" section since you incorporated the details above.
